### PR TITLE
added bionic builder host, others minor fixes

### PR DESCRIPTION
**Pull request by cisba**
- added document describing how to create an Ubuntu VirtualBox builder host
    - added document describing how to setup Gitian on the builder host
    - removed the initial make-base-vm of the Gitian image from the tail
      of Gitian setup document for Debian and Fedora, because it is already
      inside the gitian-build.py script
    - minor improvements and fixes to the signing procedure

### CODE REVIEW

File: gitian-building/gitian-building-setup-gitian-fedora.md [hunk 1]
```diff
@@ -80,23 +80,4 @@ git clone https://github.com/bitcoin-core/gitian.sigs.git
...
-This image will be copied and used every time that a build is started to
-make sure that the build is deterministic.
-Creating the image will take a while, but only has to be done once.
-
-Execute the following as user `gitianuser`:
-
-```bash
-cd gitian-builder
-git checkout 686a00ad712e30ba3a7850c16ef32f650df5b5dc # This version seems to work better than master
-bin/make-base-vm --lxc --arch amd64 --suite trusty
```
Comment (MarcoFalke) [inline gitian-building/gitian-building-setup-gitian-fedora.md]: Why is this removed? The build script can only create a bionic image and when running without the build script, this step is required.

Comment (cisba) [reply gitian-building/gitian-building-setup-gitian-fedora.md]: If you do this step here, the build script fails because the image is already present.

If you run without the build script, you find the step in gitian-building-manual.md.

Comment (cisba) [reply gitian-building/gitian-building-setup-gitian-fedora.md]: BTW, my aim was this: https://github.com/bitcoin/bitcoin/issues/12511#issuecomment-385471954

Comment (MarcoFalke) [reply gitian-building/gitian-building-setup-gitian-fedora.md]: Ah, I see. Thx.

Comment (cisba) [reply gitian-building/gitian-building-setup-gitian-fedora.md]: I understand just now...
Leaving the manual step to make-base-vm trusty is a trick to have the gitian setup for older tags.
Instead the gitian-build.py is in charge of executing make-base-vm bionic for 0.17.x and later.
:)
I will fix it the sooner the better.

**Comment by cisba** [comment]
@MarcoFalke I found the lxc bridge issue not very easy to solve, probably it is blocking for newcomers. Do you think it is better to put it in a dedicated PR to merge quickly?

**Comment by MarcoFalke** [comment]
What is the version that enables this bridge?

**Comment by cisba** [comment]
> What is the version that enables this bridge?
gitianuser@ubuntu:~$ dpkg-query -W lxc
lxc	3.0.1-0ubuntu1~18.04.2

**Comment by cisba** [comment]
Here the lxc-net service script that uses those defaults:
```
root@groucho:~# dpkg-query -S /usr/lib/x86_64-linux-gnu/lxc/lxc-net
lxc-utils: /usr/lib/x86_64-linux-gnu/lxc/lxc-net
```

**Comment by cisba** [comment]
./contrib/gitian-build.py line 49:
subprocess.check_call(['sudo', 'sed', '-i', 's/lxcbr0/br0/', '/etc/default/lxc-net'])

**Comment by MarcoFalke** [comment]
Sorry, needs rebase

**Comment by cisba** [comment]
> Sorry, needs rebase

Done

**Comment by fanquake** [comment]
Closing, now that we have migrated away from using gitian.
